Makefile

```diff
@@ -29,6 +29,9 @@ define build-tags
 		sed s/'packageVersion.*'/'packageVersion = "'$$VERSION'"'/ src/github.com/getlantern/flashlight/autoupdate.go | sed s/'!prod'/'prod'/ > src/github.com/getlantern/flashlight/autoupdate-prod.go; \
 	else \
 		echo "** VERSION was not set, using git revision instead ($(GIT_REVISION)). This is OK while in development."; \
+	fi && \
+	if [[ -z "$$HEADLESS" ]]; then \
```

Comment:
Ahh yes. I don't have a workable build environment to test Makefile so feel
free to modify it. I'm also trying to set up docker on Ubuntu VM, hopefully
I can test it tomorrow morning.
On Apr 4, 2015 12:51 AM, "Carlos Nieto" <notifications@github.com> wrote:

> In Makefile
> <https://github.com/getlantern/flashlight-build/pull/153#discussion_r27739711>
> :
>
> > @@ -29,6 +29,9 @@ define build-tags
> >  		sed s/'packageVersion.*'/'packageVersion = "'$$VERSION'"'/ src/github.com/getlantern/flashlight/autoupdate.go | sed s/'!prod'/'prod'/ > src/github.com/getlantern/flashlight/autoupdate-prod.go; \
> >  	else \
> >  		echo "** VERSION was not set, using git revision instead ($(GIT_REVISION)). This is OK while in development."; \
> > +	fi && \
> > +	if [[ -z "$$HEADLESS" ]]; then \
>
> I think this should be ! -z, since we want to check when $$HEADLESS is
> not empty.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/getlantern/flashlight-build/pull/153/files#r27739711>.
>